lib/getters/cartGetters.ts

```diff
@@ -51,9 +51,9 @@ export const getCartItemSku = (item: CartItem) =>
 
 function getTotals(cart: Cart) {
   return {
-    total: cart?.total,
-    subtotal: cart?.subtotal,
-    special: cart?.orderDiscounts ? cart?.discountedSubtotal : cart?.subtotal,
+    total: `$${cart?.orderDiscounts?.length ? cart?.discountedSubtotal : cart?.total}`,
```

Comment:
return discountedSubtotal from here and do the if statement to determine total or discountedSubtotal on the UI
also, just return the number and add formatting in the UI / component with $n

Reply:
Done